VMBackup/main/handle.py

```diff
@@ -342,6 +344,17 @@ def daemon():
                 temp_result=CommonVariables.FailedHandlerGuestAgentCertificateNotFound
                 temp_status= 'error'
                 exit_with_commit_log(temp_status, temp_result,error_msg, para_parser)
+            
+            if(freezer.file_exists == False):
+                backup_logger.log("inside if",True, "Info")
+                file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), freezer.safeFreezeFolderPath) 
+                error_msg = "safefreeze binary is missing in the following path " + str(file_path)
+                backup_logger.log("setting the error code", True, "Info")
```

Comment:
same here. why is this log statement required if the final error code is logged elsewhere?